src/cli-client/internals/token/token.go

```diff
@@ -0,0 +1,49 @@
+package token
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/canonical/oci-factory/cli-client/internals/logger"
+	"golang.org/x/term"
+)
+
+const TokenVarName = "GITHUB_TOKEN"
+
+var _accessToken = ""
+
+func readAccessToken() string {
+	envAccessToken := os.Getenv(TokenVarName)
+	if len(envAccessToken) == 0 {
+		fmt.Print("GitHub personal access token: ")
+		accessTokenBytes, err := term.ReadPassword(0)
+		if err != nil {
+			logger.Panicf("Error handling token: %v", err)
```

Comment:
Just to make sure the error message doesn't contain the token, which will be then displayed in the logs, replace any possible instances of the token inside the error message.